docs/tutorials/Soil/richards_equation.jl

```diff
@@ -148,9 +156,35 @@ t0 = FT(0)
 timeend = FT(60 * 60 * 24 * 36)
 dt = FT(100);
 
+# Now, we choose the timestepping algorithm we want to use.
+# We'll use the ARS111 algorithm with a maximum of 1 Newton iteration
+# per timestep.
+stepper = CTS.ARS111();
+ode_algo = CTS.IMEXAlgorithm(
+    stepper,
+    CTS.NewtonsMethod(
+        max_iters = 1,
```

Comment:
just a note that when we use the implicit solver, we can use a conv checker and max_iters > 1

Reply:
Discussed offline - While using fewer iterations is better for performance, it also increases the likelihood of larger errors in both our solution values and mass conservation. We'll look into this more later on, but we should test different convergence conditions and find one that gives us reasonable RMSE and mass conservation error. See #222